search-app/resources/templates/base.html

```diff
@@ -0,0 +1,35 @@
+{% extends "templates/structure.html" %}
+
+{% block title %}CMR Search{% endblock %}
+
+{% block top-nav %}
+<div class="masthead clearfix">
+  <div class="inner">
+    <h3 class="masthead-brand"><a href="/">CMR</a></h3>
+    <ul class="nav masthead-nav">
+      <li><a href="/site/search_api_docs.html">API</a></li>
```

Comment:
Hrn, I have a review feedback of my own: here there is a root path URL which is only going to be correct on local machines when CMR is running, e.g., in the REPL.

Instead, there should be a template variable here that allows for a base URL to be used, e.g.:

```html
<a href="{{ relative-root-url }}/site/XXX">XXX Text</a>
```

This will require: 
* updating the templates that have URLs with this problem
* updating the page data functions to include a `relative-root-url` key and a value taken from `public-conf`